react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JMOngoingConferenceModule.java

```diff
@@ -0,0 +1,52 @@
+package org.jitsi.meet.sdk;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.ReactActivity;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.module.annotations.ReactModule;
+
+import org.jitsi.meet.sdk.log.JitsiMeetLogger;
+
+
+/**
+ * This class implements a ReactModule and it's
+ * responsible for launching/aborting a service when a conference is in progress.
+ */
+@ReactModule(name = JMOngoingConferenceModule.NAME)
+class JMOngoingConferenceModule extends ReactContextBaseJavaModule {
+
+    public static final String NAME = "JMOngoingConference";
+
+    public JMOngoingConferenceModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+    }
+
+    @ReactMethod
+    public void launch() {
+        Context context = getReactApplicationContext();
+        ReactActivity reactActivity = (ReactActivity) getCurrentActivity();
+        JMOngoingConferenceService.launch(context, reactActivity);
```

Comment:
Do the permissions checking here, and then call launch on the service once ready.